cypress/e2e/Error.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('Error', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/thisIsaBADurl');
+  })
+
+  it('should display logo', () => {
+    cy.get(".App-logo").should('be.visible')
+  });
+
+  it('should have a home button', () => {
+    cy.get('button').should('be.visible')
```

Comment:
Does the button have any text in it? Making an assertion on what the button says could be another way to make this more robust.